CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## v6.0.0 - 2018-01-09
+
+### Breaking changes
+
+- Remove `sc` alias which conflicts with `sc.exe` (#5827)
+- Move group policy settings and enable policy controlled logging in PowerShell Core (#5791)
```

Comment:
Instead of `Move`, perhaps use `Separate`?

Reply:
That's better. I will update.